03-account-kit-manage-resources/index.ts

```diff
@@ -0,0 +1,37 @@
+// SDK exposes all you need
+import { Chains } from '@[SOMETHING]kit'
+import { Asset, RamBytes, Account, SigningRequest } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account
+    const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Buy ram
+    const buyRamEsr: SigningRequest = account.actions.buyRam(1000)
+
+    // You could then pass it to Session Kit to sign and broadcast
+    // session.transact(buyRamEsr)
+
+    // Sell ram
+    const sellRamEsr: SigningRequest = account.resources.ram.sell(RamBytes.from(1000))
```

Comment:
If `account.resources.ram` is a `UInt64`, are we going to add extra actions to it as some kind of wrapper, or should actions themselves be on the root object somehow?

Reply:
It would be cool if we could put them on the root object (although I'm not sure how to do that).